lua/nvim-tree/colors.lua

```diff
@@ -53,6 +54,7 @@ local function get_links()
   return {
     FolderName = 'Directory',
     EmptyFolderName = 'Directory',
+    OpenedFolderName = 'NvimTreeOpenedFile',
```

Comment:
this should be Directory too, because people might not want opened folders to be highlighted differently by default, users that wishes to benefit that feature will set the hl themselves :)

Reply:
Hmmm, I see. Fixing that